awx_collection/plugins/modules/credential_type.py

```diff
@@ -71,6 +71,26 @@
     state: present
     validate_certs: false
 
+- name: Custom Credential type with inputs as a YAML
+  credential_type:
+      name: Nexus
```

Comment:
The indentation looks off here. Please update this to be consistent with the other examples in this file.